### DISCUSSION TITLE
React-dropzone child dependency missing

### DISCUSSION THREAD

**Discussion by arjun2159** [react-dropzone/react-dropzone#1286] [Q&A]
I have a problem in child dependency using PNPM. Using PNPM the react-dropzone in package.json only create the react-dropzone in node_modules. It doesn't create the dropzone in node_modules. How can I fix it ?

**Comment by rolandjitsu** [comment]
@arjun2159 I'm a bit confused as to what your issue is. Could you provide a more detailed example of what you're trying to achieve and what you're experiencing?
